ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.ts

```diff
@@ -359,15 +364,15 @@ export class TissueComponent {
     }
   }
 
-  deleteSMID( array: TissueSmId[], i: number ): void {
-    array[ i ].deleted = true;
-    if (array[ i ].smIdPk) {
-      this.changeSmId('1', 'deleted', array[ i ].smIdPk, array[ i ].smIdType, array, i );
-    }
-    if (this.smIdDuplicate[ this.currentSMIDField ].has( this.createDuplicateIndex( i ) )) {
-      this.smIdDuplicate[ this.currentSMIDField ].delete( this.createDuplicateIndex( i ) );
+  deleteSMID(array: TissueSmId[], i: number): void {
+    if (array[i].smIdPk) {
+      this.changeSmId('1', 'deleted', array[i].smIdPk, array[i].smIdType, array, i);
+    } else {
+      array.splice(i, 1);
+      if (this.smIdDuplicate[this.currentSMIDField].has(this.createDuplicateIndex(i))) {
+        this.smIdDuplicate[this.currentSMIDField].delete(this.createDuplicateIndex(i));
+      }
```

Comment:
only call backend delete if data was created ( has a smIdPk) if not just delete from frontend data